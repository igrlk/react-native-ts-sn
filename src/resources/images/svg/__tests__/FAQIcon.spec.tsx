import React from 'react';
import renderer from 'react-test-renderer';

import FAQIcon from '../FAQIcon';

describe('GoBackButton', () => {
	it('match to snapshot', () => {
		const tree = renderer.create(<FAQIcon />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
