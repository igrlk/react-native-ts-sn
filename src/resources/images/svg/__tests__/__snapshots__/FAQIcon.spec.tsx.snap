// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoBackButton match to snapshot 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="36"
  bbWidth="36"
  fill="none"
  focusable={false}
  height="36"
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      undefined,
      Object {
        "opacity": 1,
      },
      Object {
        "flex": 0,
        "height": 36,
        "width": 36,
      },
    ]
  }
  vbHeight={36}
  vbWidth={36}
  width="36"
>
  <RNSVGGroup
    fill={null}
    fillOpacity={1}
    fillRule={1}
    font={Object {}}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={
      Array [
        "fill",
      ]
    }
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  >
    <RNSVGGroup
      fill={
        Array [
          0,
          4278190080,
        ]
      }
      fillOpacity={1}
      fillRule={1}
      font={Object {}}
      matrix={
        Array [
          1,
          0,
          0,
          1,
          0,
          0,
        ]
      }
      opacity={1}
      originX={0}
      originY={0}
      propList={Array []}
      rotation={0}
      scaleX={1}
      scaleY={1}
      skewX={0}
      skewY={0}
      stroke={null}
      strokeDasharray={null}
      strokeDashoffset={null}
      strokeLinecap={0}
      strokeLinejoin={0}
      strokeMiterlimit={4}
      strokeOpacity={1}
      strokeWidth={1}
      vectorEffect={0}
      x={0}
      y={0}
    >
      <RNSVGPath
        d="M35.8598 32.6006L33.3884 25.4081C34.5771 23.0052 35.2058 20.327 35.21 17.632C35.2172 12.9913 33.4216 8.60316 30.1539 5.27626C26.8857 1.94872 22.5307 0.0757423 17.8911 0.00212527C15.4603 -0.0368981 13.1016 0.410992 10.877 1.33187C8.73123 2.22013 6.80699 3.50677 5.15775 5.15602C3.50843 6.80527 2.22178 8.72951 1.3336 10.8752C0.412716 13.0999 -0.0346813 15.4598 0.00377957 17.8894C0.077256 22.5289 1.95031 26.884 5.27784 30.1523C8.59827 33.4136 12.9751 35.2086 17.6058 35.2085C17.6149 35.2085 17.6245 35.2085 17.6335 35.2085C20.3285 35.2043 23.0068 34.5756 25.4097 33.3869L32.6021 35.8583C32.8787 35.9534 33.1622 35.9997 33.4426 35.9997C34.1088 35.9997 34.7568 35.738 35.2481 35.2465C35.9463 34.5484 36.1808 33.5345 35.8598 32.6006ZM33.7389 33.7374C33.6681 33.8082 33.5165 33.9156 33.2956 33.8398L25.6742 31.221C25.5615 31.1823 25.4443 31.163 25.3274 31.163C25.1551 31.163 24.9837 31.2047 24.8283 31.2869C22.6257 32.4522 20.1366 33.0702 17.6302 33.074C9.22433 33.0876 2.271 26.2601 2.13783 17.8556C2.07082 13.627 3.67929 9.65292 6.66693 6.66528C9.65457 3.6777 13.6275 2.06952 17.8572 2.13639C26.2616 2.26963 33.0886 9.21951 33.0756 17.6287C33.0716 20.1352 32.4536 22.6243 31.2884 24.8268C31.1505 25.0875 31.1266 25.3937 31.2225 25.6727L33.8413 33.2941C33.9172 33.5148 33.8098 33.6666 33.7389 33.7374Z"
        fill={
          Array [
            0,
            4282103114,
          ]
        }
        fillOpacity={1}
        fillRule={1}
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        opacity={1}
        originX={0}
        originY={0}
        propList={
          Array [
            "fill",
          ]
        }
        rotation={0}
        scaleX={1}
        scaleY={1}
        skewX={0}
        skewY={0}
        stroke={null}
        strokeDasharray={null}
        strokeDashoffset={null}
        strokeLinecap={0}
        strokeLinejoin={0}
        strokeMiterlimit={4}
        strokeOpacity={1}
        strokeWidth={1}
        vectorEffect={0}
        x={0}
        y={0}
      />
      <RNSVGPath
        d="M17.4311 25.5057C16.8729 25.5057 16.3381 25.9965 16.3639 26.5729C16.3898 27.1511 16.8328 27.6401 17.4311 27.6401C17.9894 27.6401 18.5241 27.1492 18.4983 26.5729C18.4725 25.9946 18.0294 25.5057 17.4311 25.5057Z"
        fill={
          Array [
            0,
            4282103114,
          ]
        }
        fillOpacity={1}
        fillRule={1}
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        opacity={1}
        originX={0}
        originY={0}
        propList={
          Array [
            "fill",
          ]
        }
        rotation={0}
        scaleX={1}
        scaleY={1}
        skewX={0}
        skewY={0}
        stroke={null}
        strokeDasharray={null}
        strokeDashoffset={null}
        strokeLinecap={0}
        strokeLinejoin={0}
        strokeMiterlimit={4}
        strokeOpacity={1}
        strokeWidth={1}
        vectorEffect={0}
        x={0}
        y={0}
      />
      <RNSVGPath
        d="M17.4311 8.97705C14.5034 8.97705 12.1215 11.3589 12.1215 14.2867C12.1215 14.876 12.5993 15.3539 13.1887 15.3539C13.778 15.3539 14.2559 14.876 14.2559 14.2867C14.2559 12.5359 15.6803 11.1114 17.4311 11.1114C19.1819 11.1114 20.6063 12.5359 20.6063 14.2867C20.6063 16.0375 19.1819 17.4619 17.4311 17.4619C16.8417 17.4619 16.3639 17.9398 16.3639 18.5291V22.8818C16.3639 23.4712 16.8417 23.949 17.4311 23.949C18.0205 23.949 18.4983 23.4712 18.4983 22.8817V19.4885C20.9163 18.9932 22.7407 16.849 22.7407 14.2867C22.7407 11.3589 20.3588 8.97705 17.4311 8.97705Z"
        fill={
          Array [
            0,
            4282103114,
          ]
        }
        fillOpacity={1}
        fillRule={1}
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        opacity={1}
        originX={0}
        originY={0}
        propList={
          Array [
            "fill",
          ]
        }
        rotation={0}
        scaleX={1}
        scaleY={1}
        skewX={0}
        skewY={0}
        stroke={null}
        strokeDasharray={null}
        strokeDashoffset={null}
        strokeLinecap={0}
        strokeLinejoin={0}
        strokeMiterlimit={4}
        strokeOpacity={1}
        strokeWidth={1}
        vectorEffect={0}
        x={0}
        y={0}
      />
    </RNSVGGroup>
    <RNSVGDefs>
      <RNSVGClipPath
        fill={
          Array [
            0,
            4278190080,
          ]
        }
        fillOpacity={1}
        fillRule={1}
        matrix={
          Array [
            1,
            0,
            0,
            1,
            0,
            0,
          ]
        }
        name="clip0"
        opacity={1}
        originX={0}
        originY={0}
        propList={Array []}
        rotation={0}
        scaleX={1}
        scaleY={1}
        skewX={0}
        skewY={0}
        stroke={null}
        strokeDasharray={null}
        strokeDashoffset={null}
        strokeLinecap={0}
        strokeLinejoin={0}
        strokeMiterlimit={4}
        strokeOpacity={1}
        strokeWidth={1}
        vectorEffect={0}
        x={0}
        y={0}
      >
        <RNSVGRect
          fill={
            Array [
              0,
              4294967295,
            ]
          }
          fillOpacity={1}
          fillRule={1}
          height="36"
          matrix={
            Array [
              1,
              0,
              0,
              1,
              0,
              0,
            ]
          }
          opacity={1}
          originX={0}
          originY={0}
          propList={
            Array [
              "fill",
            ]
          }
          rotation={0}
          scaleX={1}
          scaleY={1}
          skewX={0}
          skewY={0}
          stroke={null}
          strokeDasharray={null}
          strokeDashoffset={null}
          strokeLinecap={0}
          strokeLinejoin={0}
          strokeMiterlimit={4}
          strokeOpacity={1}
          strokeWidth={1}
          vectorEffect={0}
          width="36"
          x={0}
          y={0}
        />
      </RNSVGClipPath>
    </RNSVGDefs>
  </RNSVGGroup>
</RNSVGSvgView>
`;
